public/index.html

```diff
@@ -14,8 +14,15 @@
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <!--
+    <!-- CSS only -->
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css"
+ rel="stylesheet" integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+ <!-- JavaScript Bundle with Popper -->
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/js/bootstrap.bundle.min.js"
+ integrity="sha384-A3rJD856KowSb7dwlZdYEkO39Gagi7vIsF0jrRAoQmDKKtQBHUuLZ9AsSv4jD4Xa" crossorigin="anonymous"></script>   
+ 
+ <!--
+
       Notice the use of %PUBLIC_URL% in the tags above.
```

Comment:
Using css framework will eventually facilitate designing this project but it is not professional to use both plain CSS and bootstrap at some part of the project. It makes the projects not look good. It is better to show whoever will check this project that we know what we are doing.

On this project we go with no framework. Only CSS and scss 😆

Reply:
we( me and Moise)did   discuss before about it  we are agree for this parts and the requirement doesn't mentions to use which one, it's options, but I will try to use CSS